compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangAwaitExpr.java

```diff
@@ -49,7 +48,7 @@ public String toString() {
     }
 
     @Override
-    public ExpressionNode getExpression() {
+    public BLangExpression getExpression() {
```

Comment:
As per the above comment, we don't need this change too

Reply:
Reverted with 4545a8d